gitgud/skills/newbasics/__init__.py

```diff
@@ -0,0 +1,44 @@
+from gitgud.skills.level_builder import BasicLevel
+from gitgud.skills.util import Skill
+from gitgud import operations
+from gitgud.skills.user_messages import firstcommit_status
+
+
+class FirstCommit(BasicLevel):
+    def _setup(self):
+        file_operator = operations.get_operator()
+        file_operator.destroy_repo()
+        file_operator.use_repo()
+
+    def status(self):
+        firstcommit_status(*self._get_state())
```

Comment:
I think since this status message is not currently intended to be reused, we should have the code here